rcl_lifecycle/src/rcl_lifecycle.c

```diff
@@ -67,6 +71,13 @@ rcl_lifecycle_state_machine_init(
   if (default_states) {
     rcl_lifecycle_init_default_state_machine(state_machine);
```

Comment:
It should, but the current implementation only ever returns `RCL_RET_OK`. Also it wasn't added in this pr 😛.

Anyways: 73c5f2c